mw-webapp/src/service/DayReportService.ts

```diff
@@ -19,8 +19,10 @@ export class DayReportService {
   /**
    * Get DayReportsDTO
    */
-  public static async getDayReportsDTO(): Promise<DayReportDTO[]> {
-    const dayReportsRaw = await getDocs(collection(db, PATH_TO_DAY_REPORTS_COLLECTION));
+  public static async getDayReportsDTO(dayReportUuids: string[]): Promise<DayReportDTO[]> {
+    const dayReportsRef = collection(db, PATH_TO_DAY_REPORTS_COLLECTION);
+    const dayReportsQuery = query(dayReportsRef, where("uuid", "in", dayReportUuids), orderBy("date", "desc"));
```

Comment:
does firebase integrates with typescript? It is not safe to use strings like "uuid"